kong/clustering/services/sync/rpc.lua

```diff
@@ -88,7 +88,7 @@ function _M:init_cp(manager)
       sync_status = CLUSTERING_SYNC_STATUS.NORMAL,
       config_hash = fmt("%032d", default_namespace_version),
       rpc_capabilities = rpc_peers and rpc_peers[node_id] or {},
-    }, { ttl = purge_delay, no_broadcast_crud_event = true })
+    }, { ttl = purge_delay, })
```

Comment:
Was this causing issues (having `no_broadcast_crud_event = true`, and if so what)?

I think we should at much as possible to add it there. Imagine connecting 1000 DPs and they all send pings or whatever all the time. This table is updated all the time. If we also cause worker events and clustering events it is quite a bad.

Reply:
I will add it in another PR, both full sync and incremental sync, do you think it is OK?